frontera/contrib/backends/remote/messagebus.py

```diff
@@ -93,19 +97,65 @@ def states(self):
         return None
 
 
+class BufferedMixin(object):
+    def __init__(self, *args, **kwargs):
+        super_self = super(BufferedMixin, self)
+
+        super_self.__init__(*args, **kwargs)
+
+        self._buffer = OverusedBuffer(
+            super_self.get_next_requests,
+            self._logger.debug,
+        )
+
+    def get_next_requests(self, max_n_requests, **kwargs):
+        return self._buffer.get_next_requests(max_n_requests, **kwargs)
+
+
+class AggregatePerHostMixin(object):
```

Comment:
My use case is to use more complex aggregation than just by host. I need to aggregate both by client and by host in order to achieve my goals.

So yes, GenericMessageBusBackend won't be much use in distributed mode. It was supposed to provide basic blocks to build the whatever-you-want MB backends.